src/main/java/org/openrewrite/java/testing/junitassertj/AssertNotNullToAssertThat.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.junitassertj;
+
+import org.openrewrite.AutoConfigure;
+import org.openrewrite.java.*;
+import org.openrewrite.java.tree.*;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.openrewrite.Formatting.EMPTY;
+import static org.openrewrite.Formatting.format;
+import static org.openrewrite.Tree.randomId;
+import static org.openrewrite.java.tree.MethodTypeBuilder.newMethodType;
+
+/**
+ * This is a refactoring visitor that will convert JUnit-style assertNotNull() to assertJ's assertThat().isNotNull().
+ *
+ * This visitor only supports the migration of the following JUnit 5 assertNotNull() methods:
+ *
+ * <PRE>
+ *     assertNotNull(Object actual) -> assertThat(condition).isNotNull()
+ *     assertNotNull(Object actual, String message) -> assertThat(condition).as(message).isNotNull();
+ *     assertNotNull(Object actual, Supplier<String> messageSupplier) -> assertThat(condition).withFailMessage(messageSupplier).isNotNull();
+ * </PRE>
+ */
+@AutoConfigure
+public class AssertNotNullToAssertThat extends JavaIsoRefactorVisitor {
+
+    private static final String JUNIT_QUALIFIED_ASSERTIONS_CLASS = "org.junit.jupiter.api.Assertions";
+    private static final String ASSERTJ_QUALIFIED_ASSERTIONS_CLASS_NAME = "org.assertj.core.api.Assertions";
+    private static final String ASSERTJ_ASSERT_THAT_METHOD_NAME = "assertThat";
+
+    /**
+     * This matcher uses a pointcut expression to find the matching junit methods that will be migrated by this visitor.
```

Comment:
```suggestion
     * This matcher finds the junit methods that will be migrated by this visitor.
```
All `MethodMatcher`s use pointcut expressions. Redundant to point this out in a comment outside of the `MethodMatcher` class itself.

Reply:
@tkvangorder I copied this comment from the assertTrue visitor - let me know if you would like me to implement this change for those visitors as well.

Reply:
Thanks Sofia, I already have a PR touching assertTrue/False, I will make the change there.